app/main.py

```diff
@@ -1,8 +1,31 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    our_list = []
+    for human in people:
+        person = Person(human["name"], human["age"])
+        if human.get("husband"):
+            person.husband = human["husband"]
+        if human.get("wife"):
+            person.wife = human["wife"]
```

Comment:
Similar to the previous comment, consider optimizing the assignment of `human["wife"]` to `person.wife` by checking if the wife is already in `our_list` and directly assigning the `Person` object if it exists.